assets/sass/widgets/_chosen-override.scss

```diff
@@ -0,0 +1,8 @@
+/* Override _chosen.scss styles */
+.chzn-drop > .chzn-search > input {
+  display: none;
+}
+
+.chzn-with-drop > .chzn-drop > .chzn-search > input {
+  display: inline-block;
+}
```

Comment:
do you want these overrides to apply to all components or just a certain one?

Reply:
I was specifically looking at the country chosen dropdown in the Enroll SMS view. But I think these could be applied to any chosen dropdown with search as they probably all have the same a11y issue that I'm trying to resolve.

Reply:
I would check with the other views that also use the chosen component that these styles make sense.

Reply:
These styles should go in the existing https://github.com/okta/okta-signin-widget/blob/master/assets/sass/widgets/_chosen.scss ?

Reply:
Hmm I think you brought up a good point. I'm not sure I know all the views that this chosen dropdown with search is being used in. I think for the purposes of this ticket, I should focus on the Enroll SMS view as it mentioned this view specifically and not others. If we find that there are similar a11y issues in other views, I can open a separate PR/ticket. Moving the styles under `.enroll-sms` selector for now which should limit the scope to just the Enroll SMS view. This would be in the okta-signin-widget/assets/sass/modules/_enroll.scss. Let me know if you have other ideas.

Also side note, I originally asked in the eng-sign-in-widget slack channel if I should update the _chosen.scss directly but was recommended to not make changes there and have a separate override file: https://okta.slack.com/archives/CAZH2MWEL/p1621981282066500. Seems the _chosen.scss was forked from okta-ui repo.